skeleton_scary_game.py

```diff
@@ -0,0 +1,329 @@
+import random
```

Comment:
Change the name of this file to "scary_game.py" and the test to "test_scary_game.py". You don't need the word skeleton anywhere